src/App.js

```diff
@@ -3,18 +3,18 @@ import { CssBaseline, ThemeProvider, Box } from "@material-ui/core";
 import Theme from "./Components/Theme";
 import { Switch, Route, Redirect } from "react-router-dom";
 import HomePage from "./Pages/HomePage";
-import NavigationAlt from "./Components/NavigationAlt";
+import Navigation from "./Components/Navigation";
 import CalculatorPage from "./Pages/CalculatorPage";
 import ErrorPage from "./Pages/ErrorPage";
 import AboutPage from "./Pages/AboutPage";
 import ContactPage from "./Pages/ContactPage";
 import Footer from "./Components/Footer";
 import GetInvolvedPage from "./Pages/GetInvolvedPage";
-import WhyVacatePage from "./Pages/WhyVacatePage";
-import ResourcesPage from "./Pages/ResourcesPage";
-import TheProcessPage from "./Subpages/Resources/TheProcessPage";
-import AssistancePage from "./Subpages/Resources/AssistancePage";
-// import Navigation from "./Components/Navigation";
+import GetStartedPage from "./Pages/GetStartedPage";
+// import WhyVacatePage from "./Pages/WhyVacatePage";
+// import ResourcesPage from "./Pages/ResourcesPage";
+// import TheProcessPage from "./Subpages/Resources/TheProcessPage";
+// import AssistancePage from "./Subpages/Resources/AssistancePage";
```

Comment:
We should remove all commented code unless there's a strong reason to keep it. We can always look through commit history to find old code.